Engine/Setup/BacktestingSetupHandler.cs

```diff
@@ -193,7 +193,7 @@ public bool Setup(IAlgorithm algorithm, IBrokerage brokerage, AlgorithmNodePacke
                 catch (Exception err)
                 {
                     Log.Error(err);
-                    Errors.Add("Failed to initialize algorithm: Initialize(): " + err);
+                    Errors.Add("Failed to initialize algorithm: Initialize(): " + err.Message + " StackTrace: " + err.StackTrace);
                 }
```

Comment:
I believe err.ToString() is the best way to get detailed stack traces, including inner exceptions recursively